bcda/api.go

```diff
@@ -59,10 +59,30 @@ func bulkRequest(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	syntheticIds := []string{}
```

Comment:
yeah I struggled with typing the words "beneficiaryIds" because I wasn't sure if they were the real deal or not. I'll make this change before merging.